docs/date.md

```diff
@@ -1,96 +1,398 @@
+
+**Parameters:**
+- `duration` (string or object): A `Temporal.Duration` object, a duration-like object, or a string from which to create a `Temporal.Duration`.
+- `disambiguation` (optional string): How to deal with subtractions that result in out-of-range values.
+  Allowed values are `constrain`, `balance`, and `reject`.
+  The default is `constrain`.
+
+**Returns:** a new `Temporal.Date` object which is the date indicated by `date` minus `duration`.
+
+This method subtracts `duration` from `date`, returning a date that is in the past relative to `date`.
+
+The `duration` argument can be any value that could be passed to `Temporal.Duration.from()`:
+- a `Temporal.Duration` object;
+- any object with properties denoting a duration, such as `{ days: 5 }`;
+- a string in ISO 8601 duration format, such as `P5D`.
+
+However, the `duration` argument must be in whole days.
+For example, `{ days: 5, hours: 0 }` is allowed, but `{ days: 5, hours: 12 }` will throw a `RangeError`.
+
+Some subtractions may be ambiguous, because months have different lengths.
+For example, subtracting one month from July 31 would result in June 31, which doesn't exist.
+For these cases, the `disambiguation` argument tells what to do:
+- In `constrain` mode (the default), out-of-range values are clamped to the nearest in-range value.
+- In `balance` mode, out-of-range values are resolved by balancing them with the next highest unit.
+- In `reject` mode, an addition that would result in an out-of-range value fails, and a `RangeError` is thrown.
+
+Usage example:
+```javascript
+date = Temporal.Date.from('2006-08-24');
+date.minus({years: 20, months: 4})  // => 1986-04-24
+date.minus('P14Y8D')  // => 1992-08-16
+
+date = Temporal.Date.from('2019-03-31')
+date.minus({months: 1}, 'constrain')  // => 2019-02-28
+date.minus({months: 1}, 'balance')  // => 2019-03-03
+date.minus({months: 1}, 'reject')  // => throws
+```
 
 ### date.**difference**(_other_: Temporal.Date) : Temporal.Duration
 
-Returns a new `Duration` object which is the difference between the current `Date` object and the argument `Date` value.
+**Parameters:**
+- `other` (`Temporal.Date`): Another date with which to compute the difference.
+
+**Returns:** a `Temporal.Duration` representing the difference between `date` and `other`.
+
+This method computes the difference between the two dates represented by `date` and `other`, and returns it as a `Temporal.Duration` object.
+The difference is always positive, no matter the order of `date` and `other`, because `Temporal.Duration` objects cannot represent negative durations.
+
+Usage example:
+```javascript
+date = Temporal.Date.from('2006-08-24');
+date.difference(Temporal.Date.from('2019-01-31'))  // => P12Y5M7D
+```
 
 ### date.**toString**() : string
 
-Returns an ISO 8601 string representing the current `Date` object
+**Returns:** a string in the ISO 8601 date format representing `date`.
```

Comment:
We're discussing the behavior of this method in #262

Reply:
From reading that issue it seems to only apply to toLocaleString?

Reply:
No, that issue is specifically about toString.  It has nothing to do with toLocaleString.

You can leave this as-is for the time being though since we don't yet have a concrete conclusion on #262.

Reply:
I've read over #262 several times now and can't see how it relates to toString... I must be missing something, but I'm happy to leave this until that issue is resolved.

Reply:
Gah, that's the wrong issue.  Take a look at #293. :)